File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Head.java
package superstitio.powers.masoOnly;

import superstitio.DataManager;

import static superstitio.cards.general.FuckJob_Card.BodyPart;

public class LostBodyPart_Head extends LostBodyPart {
    public static final String POWER_ID = DataManager.MakeTextID(LostBodyPart_Head.class);

    public LostBodyPart_Head() {
        super(POWER_ID);
    }


    @Override
    public BodyPart[] banedBodyPart() {
        return new BodyPart[]{BodyPart.head, BodyPart.mouth};
    }
}

File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Head.kt
package superstitio.powers.masoOnly

import superstitio.DataManager
import superstitio.cards.general.FuckJob_Card.BodyPart

class LostBodyPart_Head : LostBodyPart(POWER_ID)
{
    override fun banedBodyPart(): Array<BodyPart?>
    {
        return arrayOf(BodyPart.head, BodyPart.mouth)
    }

    companion object
    {
        val POWER_ID: String = DataManager.MakeTextID(LostBodyPart_Head::class.java)
    }
}
